lib/_debug_agent.js

```diff
@@ -7,7 +7,12 @@ const Buffer = require('buffer').Buffer;
 const Transform = require('stream').Transform;
 
 exports.start = function start() {
-  var agent = new Agent();
+  const agent = new Agent();
+
+  // SmartOS sometimes munges output from `process._rawDebug()`
+  // but Windows can't use `console.error()`.
+  // https://github.com/nodejs/node/issues/2476
+  const log = process.platform === 'win32' ? process._rawDebug : console.error;
```

Comment:
Cool, I may be in touch. (Going to focus on getting to sleep now.) BTW, it could also go the other way, I imagine, and be a "fix `console.error()` in this context for Windows` . If that happened, there would be no reason for `_rawDebug()` to exist.